### PR TITLE
Feature/m06 participant easy setup

### PR DESCRIPTION
**Pull request by Eventyret**
## What will this do?
- Setup a virtual environment using pipenv
- Copy and modify the `.env` (You still need to edit it and add your own `SECRET_KEY`)

## What will it not do?
- Create any branches for you
- Setup any databases or run any migration commands.

## Requirements for Windows
- Python installed and **added to path**

### Mac / Linux
1. Give execute permissions to file `chmod +x setup.sh`
2. run `./setup.sh`
If any errors please raise an issue and give me a tag with @eventyret

### Windows
1. Run `setup.bat` 
If any errors please raise an issue and give me a tag with @eventyret

This will help with Issue:  https://github.com/Code-Institute-Community/ci-hackathon-app/issues/70 and speed up development when using local IDE like vscode

### CODE REVIEW

File: setup.bat [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ECHO SITE_NAME="localhost" >> .env
+ECHO Installing pipenv for development use
+pip install pipenv
+CLS
+@ECHO OFF
+ECHO Installing requirements.txt
+pipenv install -r requirements.txt
+
+ECHO ============================
+ECHO Please edit .env file and add your secret key
```
Comment (stefdworschak) [inline setup.bat:16]: Could you add something for Windows like you have in the bash script to generate a random `SECRET_KEY`?

**Review by stefdworschak — APPROVED** [review]
LGTM

File: setup.bat (hunk 1 above)
Comment (Eventyret) [reply setup.bat:16]: I can try add it with a small function.  for bash we have built in openssl.
But will add something

Comment (stefdworschak) [reply setup.bat:16]: Sure. Was just a suggestion.

**Comment by stefdworschak** [comment]
@Eventyret do you have access to merge or want me to do it?

**Comment by Eventyret** [comment]
I'll push up a small fix and then I'll let you know

**Comment by Eventyret** [comment]
Added random string generation for mac and Linux @stefdworschak  
I'm unable to merge as I do not have write access to repo so you will need to merge this in.

Need to update the wiki with instructions for people too. (as i don't think I can clone and update that)

**Comment by stefdworschak** [comment]
@Eventyret I added most of the steps in the PR to the Development Wiki and will merge now

**Pull request merged**